liquibase/README.md

```diff
@@ -276,6 +282,8 @@ sets that are introduced
 
 Include the new file in `resources/liquibase-update-to-latest-from-1.9.x.xml`, it is used by integration tests (as mentioned above).
 
+The 
```

Comment:
Is this a leftover? :)

Reply:
No, `resources/liquibase-update-to-latest-from-1.9.x.xml` is used by `api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java`. That test can probably be refactored or decommissioned but that is not in scope of this ticket.

Reply:
I think by  left over he means `The` on line 285 above. 😊

Reply:
@dkayiwa true! Sorry for the confusion

Reply:
Got it...

Reply:
And got rid of it...